Create operations-manager.md (#2450)
airbytehq/airbyte: docs/career-and-open-positions/operations-manager.md

We believe in product-led growth, where we build something awesome and let our product bring the users, rather than an outbound sales engine with cold calls.

## **Description**

As our operations manager, you will have the opportunity to work directly with the co-founders, who are serial entrepreneurs. You will have mainly two focus: 

1. Helping the 2 co-founders in the day-to-day operations of the company
2. Performing the experiments that the team has thought about, but doesn’t have the time for. 

Examples of such experiments would be building the equivalent of [Segment’s Recipes](http://segment.com/recipes), or any other content / SEO initiatives. Other examples could be the organization of virtual events and hackathons. A lot of those experiments will have inspirations from growth marketing and developer advocacy. 

## **Responsibilities**

> * Lead efforts on growth experimentations

Reviewer: ```suggestion
* Lead efforts on growth experimentation
```

> * Help on day-to-day operations \(contracts, compliance, creating sanbox accounts for our connector teams, customer calls, etc.\) 

Reviewer: ```suggestion
* Help on day-to-day operations \(contracts, compliance, creating sandbox accounts for our connector teams, customer calls, etc.\) 
```

> * Help on day-to-day operations \(contracts, compliance, creating sanbox accounts for our connector teams, customer calls, etc.\) 

Reviewer 2: ```suggestion
* Help with day-to-day operations \(contracts, compliance, creating sandbox accounts for our connector teams, customer calls, etc.\) 
```

> * Help on the creation of content with the community \(case studies, etc.\)

Reviewer 2: ```suggestion
* Help with community content creation \(case studies, etc.\)
```
